Mapbox/index.html

```diff
@@ -1,254 +1,244 @@
+		let startingPosition = { lng: -79.94750, lat: 32.79250 };
+
+		// startingPosition with image at edge of strip (example 1)
+		// let startingPosition = { lng: -79.977379, lat: 32.79250 };
+
+		// startingPosition with image at edge of strip (example 2)
+		// let startingPosition = { lng: -79.988794, lat: 32.79250 };
+
+		// startingPosition with no images
+		// let startingPosition = { lng: -75.705072, lat: 32.79250 };
+
+		// Your MapBox access token
+		mapboxgl.accessToken = "YOUR_ACCESS_TOKEN";
+		let map = new mapboxgl.Map({
+			container: "map",
+			style: "mapbox://styles/mapbox/streets-v11",
+			center: startingPosition,
+			zoom: 13, // starting zoom
+		});
+		// map.showTileBoundaries = true;
+		map.addControl(new mapboxgl.NavigationControl({showCompass: false}));
+
+		// Whenever the map stops moving, load Bird.i tiles for the new location
+		map.on('moveend', function() {
+			getTiles();
+		});
+		// Whenever the map is moved, remove any existing Bird.i tiles
+		map.on('move', function() {
+			removeTiles();
+		});
+
+		// Your Bird.i token
+		const TOKEN = "YOUR_TOKEN";
+		const overlayDiv = document.getElementById('overlay');
+		let outsideBoundary = 0;
+		let fullCoverage = "";
+		/*
+		// Uncomment this line to only show an image if
+		// it fully covers the tiles you're going to load:
+		*/
+		// fullCoverage = "?fullCoverage=true";
+
+		getTiles = () => {
+			// Remove any existing tiles before loading new ones
+			removeTiles();
+			// Get current lng, lat, and zoom
+			let lng = map.getCenter().lng;
+			let lat = map.getCenter().lat;
+			let zoom = map.getZoom();
+			/*
+			// We need tiles at one higher zoom level,
+			// since mapbox vector tiles are 512x512px and Bird.i tiles are 256x256px.
+			// Also, MapBox uses continuous zoom levels (floats), whereas Bird.i
+			// only uses discrete zoom levels (ints)
+			*/
+			zoom = Math.floor(zoom+1);
+			// Show the zoom level in the bottom-left of the map
+			showZoom(zoom);
+			/*
+			// Bird.i tiles are only available within zooms 14 to 19. If you request images
+			// outside this range, you'll get an error
+			*/
+			if (zoom < 14 || zoom > 19) {
+				return;
+			}
+
+			let catalogUrl =
+				"https://catalog-api.hibirdi.com/catalog/lat/" + lat.toFixed(6) +
+				"/lng/" + lng.toFixed(6) + "/zoom/" + zoom + "/token/" + TOKEN + "/" + fullCoverage;
+
+			fetch(catalogUrl)
+			.then(response => {
+				return response.json();
+			}).then(catalogResponse => {
+				/*
+				// For this example, we are simply using the first image in the returned catalog;
+				// however, you could extend this functionality to use any of the available images,
+				// allow the user to switch between multiple images, etc
+				*/
+				console.log(catalogResponse);
+				let firstCatalogItem = catalogResponse['items'][0];
+				/*
+				// Get the x and y coordinates of the upper-left and bottom-right tiles
+				// from the available 5x5 grid
+				*/
+				const minTileX = catalogResponse['minTileX'];
```

Comment:
You could do some sweet object shorthand here if you wanted to:
```
const {minTileX, minTileY, maxTileX, maxTileY} = catalogResponse
```
(I think that's it)

Reply:
It's so beautiful! I've changed it in both of these, and I'll do the same in the others when I get to them!